_includes/scripts.html

```diff
@@ -7,11 +7,18 @@
     | push: _uswds_js %}
     {% if site.anchor_js_targets %}
       {% assign _anchor_js = '/assets/js/vendor/anchor.min.js' %}
-      {% assign _main_js = '/assets/js/main.js' %}
+      {% assign _anchor_conf_js = '/assets/js/anchor.js' %}
       {% assign _site_scripts = _site_scripts
         | push: _anchor_js
         | push: _main_js %}
     {% endif %}
+    {% if site.private_eye %}
+      {% assign _pi_js = '/assets/js/vendor/private_eye.js' %}
+      {% assign _pi_conf_js = '/assets/js/private_eye_conf.js' %}
+      {% assign _site_scripts = _site_scripts
+        | push: _pi_js
+        | push: _pi_conf_js %}
+    {% endif %}
```

Comment:
I'd be interested if anyone has thoughts on better ways to do this. I intentionally mirrored the anchor.js implementation, but I'm not sure if where I landed is the best way we could be doing this.

Reply:
Definitely feels like it could be simpler, but we can worry about that separately.